submissions/OlStaniev/html-css-popup/css/style.css

```diff
@@ -0,0 +1,203 @@
+@charset "UTF-8";
```

Comment:
Don't add autogenerated files into the homeworks folder. Only files you yourself typed in.